### PR TITLE
CTOR-920 : Refresh network-switchs-arista-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-switchs-arista-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 1]
```diff
@@ -2,48 +2,717 @@
...
+| Sensors | Net-Arista-Sensors-SNMP-custom | Contrôle les sondes                                                                                                                                                            |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Arista-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                      | Modèle de service                                 | Description                                                                                          |
+|:---------------------------|:--------------------------------------------------|:-----------------------------------------------------------------------------------------------------|
+| Packet-Errors-Generic-Id   | Net-Arista-Packet-Errors-Generic-Id-SNMP-custom   | Contrôle le pourcentage de paquets en erreur/écarté d'une interface réseau                           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:35]:
```suggestion
| Packet-Errors-Generic-Id   | Net-Arista-Packet-Errors-Generic-Id-SNMP-custom   | Contrôle le pourcentage de paquets en erreur/écartés d'une interface réseau                           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 2]
```diff
@@ -2,48 +2,717 @@
...
+| Packet-Errors-Generic-Name | Net-Arista-Packet-Errors-Generic-Name-SNMP-custom | Contrôle le pourcentage de paquets en erreur/écarté d'une interface réseau                           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:36]:
```suggestion
| Packet-Errors-Generic-Name | Net-Arista-Packet-Errors-Generic-Name-SNMP-custom | Contrôle le pourcentage de paquets en erreur/écartés d'une interface réseau                           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 3]
```diff
@@ -2,48 +2,717 @@
...
+| Packet-Errors-Global       | Net-Arista-Packet-Errors-Global-SNMP-custom       | Contrôle le pourcentage de paquets en erreur/écarté de plusieurs interfaces réseau                   |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:37]:
```suggestion
| Packet-Errors-Global       | Net-Arista-Packet-Errors-Global-SNMP-custom       | Contrôle le pourcentage de paquets en erreur/écartés de plusieurs interfaces réseau                   |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 4]
```diff
@@ -2,48 +2,717 @@
...
+| Tcpcon-Generic             | Net-Arista-Tcpcon-Generic-SNMP-custom             | Contrôle permettant de vérifier les connexions tcp                                                   |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:38]:
```suggestion
| Tcpcon-Generic             | Net-Arista-Tcpcon-Generic-SNMP-custom             | Contrôle permettant de vérifier les connexions TCP                                                   |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 5]
```diff
@@ -2,48 +2,717 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: All interfaces are ok | *interface_name*#interface.traffic.in.bitspersecond'=4212b/s;;;;' *interface_name*#interface.traffic.out.bitspersecond'=1242b/s;;;;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:429]: thresholds

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 6]
```diff
@@ -2,48 +2,716 @@
...
+* **Net-Arista-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Arista-SNMP-custom" label="Net-Arista-SNMP-custom">
+
+| Service Alias | Service Template               | Service Description                                                                                                                                  |
+|:--------------|:-------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cpu           | Net-Arista-Cpu-SNMP-custom     | Check the rate of utilization of CPU for the equipment. This check can give the average CPU utilization rate and the rate per CPU for multi-core CPU |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:23]:
```suggestion
| Cpu           | Net-Arista-Cpu-SNMP-custom     | Check the rate of utilization of the CPU for the resource. This check can give the average CPU utilization rate and the rate per CPU for multi-core CPU |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 7]
```diff
@@ -2,48 +2,716 @@
...
+| Memory        | Net-Arista-Memory-SNMP-custom  | Check the rate of the utilization of memory                                                                                                          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:24]:
```suggestion
| Memory        | Net-Arista-Memory-SNMP-custom  | Check the rate of the utilization of the memory                                                                                                          |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 8]
```diff
@@ -2,48 +2,716 @@
...
+| Sensors       | Net-Arista-Sensors-SNMP-custom | Check sensors                                                                                                                                        |
+
+> The services listed above are created automatically when the **Net-Arista-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias              | Service Template                                  | Service Description                                                                          |
+|:---------------------------|:--------------------------------------------------|:---------------------------------------------------------------------------------------------|
+| Packet-Errors-Generic-Id   | Net-Arista-Packet-Errors-Generic-Id-SNMP-custom   | Check packets on errors/discards of a network interface                                      |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:34]:
```suggestion
| Packet-Errors-Generic-Id   | Net-Arista-Packet-Errors-Generic-Id-SNMP-custom   | Check packets in error/discarded from a network interface                                      |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 9]
```diff
@@ -2,48 +2,716 @@
...
+| Packet-Errors-Generic-Name | Net-Arista-Packet-Errors-Generic-Name-SNMP-custom | Check packets on errors/discards of a network interface                                      |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:35]:
```suggestion
| Packet-Errors-Generic-Name | Net-Arista-Packet-Errors-Generic-Name-SNMP-custom | Check packets in error/discarded from a network interface                                     |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 10]
```diff
@@ -2,48 +2,716 @@
...
+| Packet-Errors-Global       | Net-Arista-Packet-Errors-Global-SNMP-custom       | Check packets on errors/discards of multiple network interfaces                              |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:36]:
```suggestion
| Packet-Errors-Global       | Net-Arista-Packet-Errors-Global-SNMP-custom       | Check packets in error/discarded from multiple network interfaces                              |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 11]
```diff
@@ -2,48 +2,716 @@
...
+| Tcpcon-Generic             | Net-Arista-Tcpcon-Generic-SNMP-custom             | Check current tcp connections                                                                |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:37]:
```suggestion
| Tcpcon-Generic             | Net-Arista-Tcpcon-Generic-SNMP-custom             | Check current TCP connections                                                                |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 12]
```diff
@@ -2,48 +2,716 @@
...
+| Traffic-Generic-Id         | Net-Arista-Traffic-Generic-Id-SNMP-custom         | Check the bandwidth of the interface. For each checks the name of the interface will appear  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:38]:
```suggestion
| Traffic-Generic-Id         | Net-Arista-Traffic-Generic-Id-SNMP-custom         | Check the bandwidth of the interface. For each check the name of the interface will appear  |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 13]
```diff
@@ -2,48 +2,716 @@
...
+| Traffic-Generic-Name       | Net-Arista-Traffic-Generic-Name-SNMP-custom       | Check the bandwidth of the interface. For each checks the name of the interface will appear  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:39]:
```suggestion
| Traffic-Generic-Name       | Net-Arista-Traffic-Generic-Name-SNMP-custom       | Check the bandwidth of the interface. For each check the name of the interface will appear  |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 14]
```diff
@@ -2,48 +2,716 @@
...
+| Traffic-Global             | Net-Arista-Traffic-Global-SNMP-custom             | Check the bandwidth of interfaces. For each checks the name of the interface will appear     |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:40]:
```suggestion
| Traffic-Global             | Net-Arista-Traffic-Global-SNMP-custom             | Check the bandwidth of interfaces. For each check the name of the interface will appear     |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md [hunk 15]
```diff
@@ -2,48 +2,716 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All interfaces are ok | *interface_name*#interface.traffic.in.bitspersecond'=4212b/s;;;;' *interface_name*#interface.traffic.out.bitspersecond'=1242b/s;;;;
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-switchs-arista-snmp.md:428]:
```suggestion
OK: All interfaces are ok | *interface_name*#interface.traffic.in.bitspersecond'=4212b/s;80;90;;' *interface_name*#interface.traffic.out.bitspersecond'=1242b/s;80;90;;
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**